src/java.base/share/classes/com/sun/crypto/provider/CipherBlockChaining.java

```diff
@@ -59,6 +59,11 @@ class CipherBlockChaining extends FeedbackCipher  {
     // variables for save/restore calls
     private byte[] rSave = null;
 
+    // chunkSize is multiple of block size, but otherwise fairly arbitrary
+    // Should be large enough to provide intrinsic with optimization
```

Comment:
Typo: `Should be small enough`

Reply:
I believe the original wording is correct: the more payload the intrinsic gets the better (eg. the intrinsic parallelizes decryption, at the cost of some initial setup. More payload amortizes the setup cost). But cannot be so large that the intrinsic locks up the VM (preventing GC), encrypting gigabytes.

Reply:
But it takes a few calls before hotspot switches to the intrinsic, so it can't be too large. I think we should include this logic explanation (the intrinsic parallelizes decryption) in the comment to make it clear what we are doing here.

Reply:
Maybe focus that this is triggering the optimization and not about the data size.
```
// chunkSize is a multiple of block size and used to divide up
// input data to trigger the intrinsic.
```

Reply:
@ascarpino Thanks! I was 30 seconds from pushing my rewording, but will take yours :D